.github/workflows/build_dev_docs.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
why do we need 2 new workflows if the only difference is on the mike command?
I think you can do this with a condition on the step.